src/_booz_xform/bindings.cpp

```diff
@@ -311,12 +322,12 @@ quantity is zero so the array will have size 0 x 0.)")
     .def_readonly("gmnc_b", &Booz_xform::gmnc_b, R"(
 (2D float array of size mnboz x ns_b, output) cos(m * theta_B - n *
 zeta_B) Fourier modes (with respect to Boozer coordinates) of the
-Jacobian of (psi, theta_B, zeta_B) coordinates.)")
+Jacobian of (s, theta_B, zeta_B) coordinates.)")
```

Comment:
Just one comment: the previous definition for gmnc_b is what is actually used. See 
https://github.com/hiddenSymmetries/booz_xform/blob/main/src/_booz_xform/surface_solve.cpp#L262
and 
https://github.com/hiddenSymmetries/booz_xform/blob/main/src/_booz_xform/surface_solve.cpp#L315